server/data/quotes.json

```diff
@@ -663,6 +663,10 @@
     "author": "Victor Hugo",
     "sentence": "To rescue from oblivion even a fragment of a language which men have used and which is in danger of being lost—that is to say, one of the elements, whether good or bad, which have shaped and complicated civilization—is to extend the scope of social observation and to serve civilization."
   },
+  {
+    "author": "Victor Hugo",
+    "sentence": "To learn to read is to light a fire, every syllable that is spelling out is a spark."
```

Comment:
This quote isn't acceptable. we need quotes based on "learning a foreign language" or close to this.